packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -24,6 +24,13 @@ describe('RestServer (integration)', () => {
     await server.stop();
   });
 
+  it('does not throw an error when stopping an app that has not been started', async () => {
+    const app = new Application();
+    app.component(RestComponent);
+    const server = await app.getServer(RestServer);
```

Comment:
These three lines can be simplified as follows:

```ts
const server = await givenAServer();
```

I'll make the change for you to speed up the process.